src/devices/DCMotor/DCMotorWithStartStop.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.DCMotor
+{
+    /// <summary>
+    /// Direct current (DC) motor with Start/Stop
+    /// </summary>
+    public class DCMotorStartStop : DCMotor
+    {
+        private DCMotor _inner;
+        private bool _stopped = false;
+        private double _speed;
+
+        /// <summary>
+        /// Constructs instance with added Start() and Stop() as additional protection
+        /// </summary>
+        /// <param name="innerMotor">Crate DCMotor instance</param>
+        public DCMotorStartStop(DCMotor innerMotor)
+            : base(null, false)
+        {
+            _inner = innerMotor;
+            _speed = innerMotor.Speed;
+        }
+
+        /// <summary>
+        /// Enable motor operation.
+        /// </summary>
+        public void Start()
+        {
+            _stopped = false;
+            _inner.Speed = _speed;
+        }
+
+        /// <summary>
+        /// Disable motor operation.
+        /// </summary>
+        public void Stop()
+        {
+            _stopped = true;
+            _inner.Speed = 0.0;
+        }
+
+        /// <summary>
+        /// Gets or sets the speed of the motor. Range is -1..1 or 0..1 for 1-pin connection.
+        /// 1 means maximum speed, 0 means no movement and -1 means movement in opposite direction.
+        /// </summary>
+        public override double Speed
+        {
+            get => _speed;
+            set
+            {
+                _speed = value;
+                if (!_stopped)
+                {
+                    _inner.Speed = _speed;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Get or Set motor status.
+        /// </summary>
+        public bool Enabled
+        {
+            get => !_stopped;
+            set
+            {
+                if (value)
+                {
+                    Start();
+                }
+                else
+                {
+                    Stop();
```

Comment:
You should also implement Dispose and Dispose of the inner motor (with shouldDispose in the constructor set to true by default)